### PR TITLE
Harshad_Assignment_25_Done_10mins

### PR DESCRIPTION
**Pull request by HarshadBJun20**
-> Assignment 25 Done.
-> Program screenshot of output:
image

Please review.

### CODE REVIEW

File: src/harshadBJun20/assignment25/SumOfNumbersInString.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+public class SumOfNumbersInString {
+	
+	void calSumOfNumbers(String str) {
+		int sumOfDigit=0;
+		int digit=0;
+		for(int index=0;index<str.length();index++) {
+			if(!Character.isLetter(str.charAt(index))) {
+				String charValue = String.valueOf(str.charAt(index));
+				digit = Integer.parseInt(charValue);
+				sumOfDigit +=digit;
```
Comment (AasthaJJun20) [inline src/harshadBJun20/assignment25/SumOfNumbersInString.java]: Hi Harshad,
We don't need to initialize variables -> digit and charValue 
You can directly check the condition:
	if(Character.isDigit(str.charAt(index)))
and then add the number in sumOfDigit 
	sumOfDigit += Character.getNumericValue(str.charAt(index)));

**Review by AasthaJJun20 — CHANGES_REQUESTED** [review]
Please remove the variables : digit and charValue

File: src/harshadBJun20/assignment25/SumOfNumbersInString.java (hunk 1 above)
Comment (KrishnaKTechnocredits) [reply src/harshadBJun20/assignment25/SumOfNumbersInString.java]: @HarshadBJun20 work on suggestion please

Comment (HarshadBJun20) [reply src/harshadBJun20/assignment25/SumOfNumbersInString.java]: Hi @AasthaJJun20,
Yes you are right. Thanks for the suggestions. I have made the necessary changes. Please review.

**Review by AasthaJJun20 — APPROVED** [review]
Code looks good to me!

**Review by KrishnaKTechnocredits — APPROVED** [review]
Code looks good to me.

**Pull request merged**